src/index.html

```diff
@@ -3,11 +3,12 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <title>React Webpack</title>
+        <title>Events Manager</title>
```

Comment:
You can also look into using React-Helmet so that you can have dynamic titles for the different app pages

Reply:
will do